tests/unit/channel_tests.py

```diff
@@ -522,7 +522,9 @@ def test_close_basic_cancel_called(self):
         self.obj._consumers['abc'] = None
         with mock.patch.object(self.obj, 'basic_cancel') as basic_cancel:
             self.obj.close()
-            basic_cancel.assert_called_once_with(consumer_tag='abc', nowait=True)
+            # this is actually not necessary but Pika currently cancels
+            # every consumer before closing the channel
+            basic_cancel.assert_called_once_with(consumer_tag='abc', nowait = True)
```

Comment:
This is super nit-picky, but typical Python style for parameter default assignment is to not use spaces.